consensus/clique/src/main/java/org/hyperledger/besu/consensus/clique/blockcreation/CliqueMiningCoordinator.java

```diff
@@ -33,6 +41,30 @@ public CliqueMiningCoordinator(
     this.miningTracker = miningTracker;
   }
 
+  @Override
+  protected void inSyncChanged(final boolean inSync) {
+    synchronized (this) {
+      final VoteTally validatorProvider =
+          miningTracker
+              .getProtocolContext()
+              .getConsensusState(CliqueContext.class)
+              .getVoteTallyCache()
+              .getVoteTallyAfterBlock(blockchain.getChainHeadHeader());
+      final boolean isValidator =
+          getCoinbase()
```

Comment:
do we want to be careful here? The coinbase is not necessarily this node's ID (maybe in our code base, but the concepts are different) - it'd be nice to see this comparing nodeKey against validator list

Reply:
Done. I created an `isSigner` method in CliqueHelpers because there was already a very close implementation and it used the node's ID